content/reference/user/artifact_conf/index.md

```diff
@@ -58,7 +56,7 @@ You can also upload artifacts programmatically with our REST API.
 
 You'll need:
 * an [API token]({{< ref "../../admin/api_tokens" >}}) with at least the `Artifacts` permission
-* the Artifact's ID you can copy from the WebUI.
+* the Artifact's ID, which you can copy from the WebUI.
```

Comment:
, which can be copied from the artifacts table.

Reply:
Always prefer the active voice!